File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int userinput;
    do {
        userinput = get_int("Please input a positive integer: ");
    } while(userinput < 1);

    int space = userinput - 1;
    for(int i = 1; i <= userinput; i++){
        for(int j = 0; j < space; j++){
            printf(" ");
        }
        space--;
        for(int j = 0; j < i; j++){
            printf("#");
        }
        printf("  ");
        for(int j = 0; j < i; j++){
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario.py
height = 0;
while(height < 1 or height > 8):
    height = int(input("Positive integer between 1 and 8: "))

space = height - 1

i = 1
while(i <= height):
    for j in range(space):
        print(" ", end="")

    for j in range(i):
        print("#" , end="")

    print("  ", end="")

    for j in range(i):
        print("#", end="")
    print("")
    i += 1
    space -= 1
